cpp/include/legate_dataframe/search.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <legate.h>
+#include <legate_dataframe/core/column.hpp>
+#include <legate_dataframe/core/library.hpp>
+
+namespace legate::dataframe {
+
+namespace task {
+
+class ContainsTask : public Task<ContainsTask, OpCode::Contains> {
+ public:
+  static void cpu_variant(legate::TaskContext context);
+  static void gpu_variant(legate::TaskContext context);
```

Comment:
Would appreciate if you can mask the gpu variant in the macro even though its not tested in CI yet.

Reply:
Good catch, I forgot. I think also probably in the copying will add it here for both.